mne/viz/evoked.py

```diff
@@ -1081,9 +1023,12 @@ def whitened_gfp(x, rank=None):
                               hline=[-1.96, 1.96], show=False)
     else:
         for ((ch_type, picks), ax) in zip(picks_list, axes_evoked):
-            ax.plot(times, evokeds_white[0].data[picks].T, color='k')
+            ax.plot(times, evokeds_white[0].data[picks].T, color='k',
+                    lw=0.5)
             for hline in [-1.96, 1.96]:
-                ax.axhline(hline, color='red', linestyle='--')
+                ax.axhline(hline, color='red', linestyle='--', lw=2)
+            ax.set(title='{0} ({1} channels)'.format(titles_[ch_type],
+                                                     len(picks)))
```

Comment:
this is a bug / consistency fix (the one I show with a script in a previous comment)